### PR TITLE
Broken Authentication Lab

### PR DESCRIPTION
**Pull request by kUSHAL0601**
Created Lab for Broken Authentication, where login can be manipulated because of the use of cookies

### CODE REVIEW

**Comment by adeyosemanputra** [comment]
@kUSHAL0601 it's conflict can you resolve it?

**Comment by kUSHAL0601** [comment]
The conflicts need change in the master branch of this repo too... I don't have write access to this repo. It will be easier if you could do it...

**Comment by kingthorin** [comment]
> The conflicts need change in the master branch of this repo too... I don't have write access to this repo. It will be easier if you could do it...

Actually your PR needs to be updated to account for the latest changes to master.

**Comment by kUSHAL0601** [comment]
@adeyosemanputra could you have a look and approve

**Review by adeyosemanputra — CHANGES_REQUESTED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -2,9 +2,7 @@ asgiref==3.2.7
 Django==3.0.14
 pytz==2020.1
 sqlparse==0.3.1
+secrets
```
[the remark this answers is not in the document]
Comment (kUSHAL0601) [reply requirements.txt:5]: Sure.

[the remark this answers is not in the document]
Comment (kUSHAL0601) [reply requirements.txt]: Removed secrets since it is an inbuilt module for python

**Comment by kUSHAL0601** [comment]
@adeyosemanputra could you have a look once

**Comment by adeyosemanputra** [comment]
sure @kUSHAL0601 , @pwned-17 will working on it.

**Comment by pwned-17** [comment]
Hey @kUSHAL0601 I have merged it and it's working great , I guess we can add your lab as a second lab for Broken Authentication Lab .

**Comment by adeyosemanputra** [comment]
@@all-contributors please add @kUSHAL0601 for code

**Pull request merged**